include/ola/base/Macro.h

```diff
@@ -63,6 +63,30 @@
 #endif  // __GNUC__
 
 /**
+ * @def OLA_FALLTHROUGH
+ * @brief Mark switch cases as fallthrough when required
+ *
+ * @examplepara
+ *   @code
+ *   switch (cond) {
+ *     case 'foo':
+ *       doFoo();
+ *       // after foo fallthrough to bar because $REASON
+ *       OLA_FALLTHROUGH
+ *     case 'bar':
+ *   @endcode
+ */
+#ifdef __GNUC__
+#if __GNUC__ >= 7
+#define OLA_FALLTHROUGH __attribute__ ((fallthrough));
```

Comment:
Can you add a future todo/comment for why we aren't using the [[fallthrough]] variant please (as we're still on C++98).